### PR TITLE
hack: fix OWASP configuration (temp solution to get builds passing)

### PR DESCRIPTION
**Pull request by akomisarek**
### JIRA link (if applicable) ###
N/A

### Change description ###

Fix config for owasp. 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: service/build.gradle [hunk 1]
```diff
@@ -16,7 +16,24 @@ plugins {
 apply plugin: 'org.springframework.boot'
 apply plugin: 'io.spring.dependency-management'
 apply plugin: 'info.solidsoft.pitest'
-apply plugin: 'org.owasp.dependencycheck'
+
+subprojects {
```
Comment (timja) [inline service/build.gradle:20]: isn't this already in allprojects, why is this needed again?

Comment (akomisarek) [reply service/build.gradle]: I was playing with the config, I removed that now. Seems to work locally as well.

File: service/build.gradle [hunk 2]
```diff
@@ -16,7 +16,20 @@ plugins {
 apply plugin: 'org.springframework.boot'
 apply plugin: 'io.spring.dependency-management'
 apply plugin: 'info.solidsoft.pitest'
-apply plugin: 'org.owasp.dependencycheck'
+
+// https://jeremylong.github.io/DependencyCheck/dependency-check-gradle/configuration.html
+dependencyCheck {
+  // Specifies if the build should be failed if a CVSS score above a specified level is identified.
```
Comment (banderous) [inline service/build.gradle:22]: this comment is obsolete

Comment (akomisarek) [reply service/build.gradle]: Thanks, removed and cleaned up a bit so it shows less changes.
